homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -27,34 +27,41 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+type BMWConfigEntry = ConfigEntry[BMWDataUpdateCoordinator]
+
+
 class BMWDataUpdateCoordinator(DataUpdateCoordinator[None]):
     """Class to manage fetching BMW data."""
 
     account: MyBMWAccount
+    read_only: bool
```

Comment:
```suggestion
```

Reply:
Can you help me in 2 sentences what the difference is?

Reply:
Its the same, but now the type is declared at where it is defined rather than define in the constructor and type outside